### PR TITLE
BB-6: update lifecycle bucket processor

### PR DESCRIPTION
**Pull request by alexanderchan-scality**

### CODE REVIEW

File: extensions/lifecycle/tasks/LifecycleTask.js [hunk 1]
```diff
@@ -780,20 +788,22 @@ class LifecycleTask extends BackbeatTask {
...
-                .addContext({
-                    origin: 'lifecycle',
-                    ruleType: 'expiration',
-                    reqId: log.getSerializedUids(),
-                })
-                .setAttribute('target.owner', bucketData.target.owner)
-                .setAttribute('target.bucket', bucketData.target.bucket)
-                .setAttribute('target.key', obj.Key)
-                .setAttribute('details.lastModified', obj.LastModified);
+                  .addContext({
```
Comment (rachedbenmustapha) [inline extensions/lifecycle/tasks/LifecycleTask.js]: This looks like inconsistent indentation causing more changes than expected.

File: extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js [hunk 2]
```diff
@@ -246,40 +315,51 @@ class LifecycleBucketProcessor {
...
-        let consumerReady = false;
+    _setupConsumer(cb) {
         this._consumer = new BackbeatConsumer({
             zookeeper: {
                 connectionString: this._zkConfig.connectionString,
             },
             kafka: {
                 hosts: this._kafkaConfig.hosts,
-                backlogMetrics: this._kafkaConfig.backlogMetrics,
+                // backlogMetrics: this._kafkaConfig.backlogMetrics,
```
Comment (rachedbenmustapha) [inline extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js:354]: Is this not expected to work?

Comment (alexanderchan-scality) [reply extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js:354]: `BackbeatConsumer` at the moment does not have the `backlogMetrics` implemented, and, in my opinion, is outside the scope of this PR

**Review by nicolas2bert — APPROVED** [review]

**Review by rachedbenmustapha — APPROVED** [review]

**Pull request merged**